src/ray/protobuf/common.proto

```diff
@@ -139,7 +139,6 @@ message RayException {
 /// run some task or actor.
 message RuntimeEnv {
   repeated string uris = 1;
```

Comment:
It looks like this isn't used in any of the other protos now. Can we remove it? Or is it used elsewhere? Btw, is it possible for the URIs to also be only in python or do they need to be accessible in C++?

Reply:
Let me think about this -- @iycheng, do you happen to know the answer offhand?

Reply:
Currently it seems to be used in the JobConfig proto in gcs.proto, but nowhere else